src/main/java/org/parabot/core/asm/wrappers/Setter.java

```diff
@@ -25,7 +25,7 @@ public Setter(final String fieldLocation, String into, final String fieldName, f
         this.into = ASMUtils.getClass(into);
         this.field = ASMUtils.getField(this.fieldLocation, fieldName, fieldDesc);
         this.name = methodName;
-        this.desc = (desc == null) ? this.field.desc : desc;
+        this.desc = (desc == null) && this.field != null ? this.field.desc : desc;
```

Comment:
What does this change do?

Reply:
In case of a hook where <desc> is not defined and the <field> is broken it would try to access field.desc which throws an exception. With this fix, it will be able to create the Setter object so but will still crash when initiating the Adapter so I can catch it there.
It makes things easier because before the code could crash in 2 locations.

Reply:
But it still crashes, correct?

Reply:
Yup, when trying to initiate the Adapter it crashes. I've tested all code by intentionally breaking all 5 types of wrappers.